shopify/usage-subscription-template/api/models/shopifyShop/actions/uninstall.ts

```diff
@@ -5,14 +5,16 @@ import {
   save,
   ActionOptions,
   ShopifyShopState,
-  UninstallShopifyShopActionContext,
 } from "gadget-server";
 import { trialCalculations } from "../../../../utilities";
 
-/**
- * @param { UninstallShopifyShopActionContext } context
- */
-export async function run({ params, record, logger, api, connections }) {
+export const run: ActionRun = async ({
+  params,
+  record,
+  logger,
+  api,
+  connections,
+}) => {
   transitionState(record, {
     from: ShopifyShopState.Installed,
     to: ShopifyShopState.Uninstalled,
```

Comment:
I see this for line 37
![CleanShot 2024-11-07 at 15 07 56@2x](https://github.com/user-attachments/assets/dcf5575f-b792-46a2-88b8-b24dd9cb2d44)